src/client/src/.tests/components/chatpage.test.js

```diff
@@ -117,13 +122,13 @@ describe('<ChatPage />', () => {
 	})
 	describe('<ChatPage /> Functionality Tests', () => {
 		it('Should update joined rooms', () => {
-			const EXPECTED_DATA = {roomId: DUMMY_ROOM, usernames: DUMMY_USER}
+			const EXPECTED_DATA = {roomId: DUMMY_ROOM, username: DUMMY_USER}
 			suite.Component.instance().updateJoinedRooms(EXPECTED_DATA)
 			const USERROOM_ROOMID = suite.Component.state(USERROOMS)[FIRST_INDEX].roomId
 			const USERROOM_USERNAMES = suite.Component.state(USERROOMS)[FIRST_INDEX].usernames
```

Comment:
Do we want to use `usernames` here, or `username`?

Reply:
These tests will be updated when I add functionality for multiple users of a single room.